### PR TITLE
Create meetup reflection post for WildGinger23

### PR DESCRIPTION
**Pull request by WildGinger23**
This is my meetup cultural immersion reflection post. It was an interesting experience.

### CODE REVIEW

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+### Details
+
+Have you heard about USE Method (Utilization - Saturation - Errors), RED (Rate - Errors - Duration) or Golden Signals (Latency - Traffic - Errors - Saturations)
+In this presentation we will talk briefly about these different, but similar “focuses” and discuss how we can apply them to the data infrastructure performance analysis troubleshooting and monitoring.
+We will use MySQL as example but most of the talk will apply to other database technologies as well.
+
+### About the speaker, Peter Zaitsev:
+
+Peter Zaitsev co-founded Percona and assumed the role of CEO in 2006. As one of the foremost experts on MySQL strategy and optimization, Peter leveraged both his technical vision and entrepreneurial skills to grow Percona from a two-person shop to one of the most respected open source companies in the business. With over 140 professionals in 30 plus countries, Peter’s venture now serves over 3000 customers – including the “who’s who” of internet giants, large enterprises and many exciting startups. Percona was named to the Inc. 5000 in 2013, 2014, 2015 and 2016.
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: Peter is a world-renowned expert on MySQL, and Percona is a company you call if you have a problem with a mission critical MySQL database at a large company.  Very cool that you got to hear one of his talks!

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+There was a representitive for LifeOmic who discussed a few projects before the formal presentation. He talked about gremlin and another member of the group mentioned that a plugin by NeoFrage (sp) translates gremlin into Cypher.
+Peter discussed a conference that I didn't quite get the name of it
+There was a member that was discussing the startup AEO Analytics that mainly works with data analytics. The rep from LifeOmic was curious about investors and employees. 
+The screens at the front that once had the wifi login were changed to two video feeds of the room before us.
+
+## Introduction 
+
+Chris started us off. Announcements DevosDay raleigh at the McKimmon center (NCSU) triangledevops twitter discount code (Chris is a speaker) Food by Hayes.
+Hayes announced two open positions, Oracle cloud experience preferred with first, and Linux. Hayes is the only woman in a room of 20-30.
+Chris says this is a monthly meeting. Asks if anyone is looking for a job, asks if anyone is hiring. SolarWinds and CargoSphere is hiring. HiringThing full remote Python and Ruby 2. LifeOmic is hiring.
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: Have a job, need a job: one of the most useful aspects of meetups, professionally speaking.

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+## Presenter 7:15 - 8:15
+
+Performance Analysis and Troubleshooting Methodologies for Databases. Peter Zaitsev of CEO of Percona is hiring. 
+
+"Who here has worked with databases?"
+"Who here hates databases?"
+"You have to hate databases, its like your wife, I mean you have to love her but..."
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: Hmm.  Not sure I'd agree with that.  Seems like a pretty clumsy intro, with needlessly misogynistic overtones.

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+USE is proactive rather than reactive.
+
+Prometheus and more
+
+## Takeaways
+
+###How did it go?
+
+I think it went pretty well. As expected a lot of the details were over my head, but I was able to generally follow along and understand what the speaker was talking about. The atmosphere was welcoming and they provided free pizza. One thing that kind of made me step back and think was when the CEO of Percona made a joke about hating his wife. Diversity was on my mind after our discussion and I looked around the room and didn't see any women. It felt good to be exposed to this community and it was interesting to see how many older men were in attendance.
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: I'm glad you caught this.  Systems and database administration seem to me to be some of the last holdouts of 'old school' programming culture, which can sometimes be shielded from the positive changes happening in more charismatic portions of the industry.

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+###What did you learn?
+
+I don't think I learned much of value for INLS 523 Databases, however I didn't really expect to. I focused on trying to follow along in the discussion of the difference between Development and Operations, because I felt that that was a simple binary perspective I could generally understand. Some of the examples that the speaker gave made what he was discussing more intelligible.
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: I'm glad you got a sense of what a high performance database engineer might do, but you're right that this won't be directly applicable to 523.  The databases you're working with will be tiny, and you'll focus on learning SQL, not really managing large existing databases.  But hopefully this is a glimpse of where you might go if you work on the back end of a database at a large institution.

File: _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+
+###Would you find value in this kind of event after class?
+
+I think I really would. I thought back to my experience digitizing reports for the NC Cooperative Extension Service. In every single one of North Carolina's 100 counties Farm Extension Agents and Home Demonstration Agents were assigned to hold meetings to teach people about farming, canning, and useful life skills in a community setting. What this meetup has really illuminated for me is the value of community learning. While there are bountiful resources online, in the future if I ever want to learn about a specific type of programming I will be sure to investigate engaging with the local community.
```
Comment (eah13) [inline _posts/WildGinger23/2018-06-14-WildGinger23-Meetup-Reflection.md]: Good thoughts!  If you keep that focus on the value of community, you'll do well.  Think about how you might get together or find people with interests even more closely aligned with your own.

**Comment by NobrineCalc** [comment]
This is the story I talked about concerning privacy:
https://www.cbsnews.com/news/how-to-shop-online-without-being-tracked/

**Pull request merged**